apps/sms/services/js/messaging/messaging_service.js

```diff
@@ -182,4 +186,6 @@ exports.MessagingService = BridgeServiceMixin.mixin(
   { methods: METHODS }
 );
 
+MessagingService.init();
```

Comment:
__note:__ that's why I've added that ```if ('importScripts' in self)``` in my patch :) So that ```init``` is called on-demand in tests and all internal things can be stubbed in advance, on contrary in this case ```init``` will be called once we load file in the test suite.